buildSrc/build.gradle

```diff
@@ -113,6 +113,18 @@ dependencies {
     module("org.bouncycastle:bcutil-jdk15on") {
       replacedBy("org.bouncycastle:bcutil-jdk15to18", "the jdk15on artifacts are not supported anymore")
     }
+    module("org.bouncycastle:bcprov-jdk15to18") {
```

Comment:
@mwilso3 I think these artifacts are still supported:
- https://mvnrepository.com/artifact/org.bouncycastle/bcpg-jdk15to18/1.78.1
- https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk15to18/1.78.1
  ...

AFAIK it was a brief period of time those were not published.